src/ui/server/controllers/redirects/index.ts

```diff
@@ -1,15 +1,28 @@
 import generateRedirectUrl from '../../helpers/generate-redirect-url';
+import isValidReqOriginalUrl from '../../helpers/is-valid-req-original-url';
+import { ROUTES } from '../../constants';
 import { Request, Response } from '../../../types';
 
+const { PROBLEM_WITH_SERVICE } = ROUTES.INSURANCE;
+
 /**
  * get
- * Redirect an MVP_INSURANCE_ROOT URL to an INSURANCE_ROOT URL
+ * Redirect an MVP_INSURANCE_ROOT URL to an INSURANCE_ROOT URL if req.originalUrl is valid
+ * Redirects to PROBLEM_WITH_SERVICE if req.originalUrl is invalid
  * @param {Express.Request} Express request
  * @param {Express.Response} Express response
  * @returns {Express.Response.redirect}
  */
 const get = (req: Request, res: Response) => {
-  const newUrl = generateRedirectUrl(req.originalUrl);
+  const { originalUrl } = req;
+
+  const validUrl = isValidReqOriginalUrl(originalUrl);
+
+  if (!validUrl) {
+    return res.redirect(PROBLEM_WITH_SERVICE);
```

Comment:
Might be good add a console.error I think.

Reply:
Not a blocker (so we can progress PRs), can we change this to the following?:

```ts
Invalid original URL detected in "redirects" controller
```

So that it's more descriptive, otherwise it reads like `/redirects` is route/URL which is not the case

Happy to be done in the next PR.